index.yaml

```diff
@@ -751,16 +751,18 @@ entries:
       catalog.cattle.io/release-name: rancher-alerting-drivers
     apiVersion: v2
     appVersion: 1.16.0
-    created: "2021-06-24T21:41:35.656007-07:00"
+    created: "2021-07-30T09:10:54.326123-07:00"
     dependencies:
-    - name: prom2teams
+    - condition: prom2teams.enabled
+      name: prom2teams
       repository: file://./charts/prom2teams
     - condition: sachet.enabled
       name: sachet
       repository: file://./charts/sachet
     description: The manager for third-party webhook receivers used in Prometheus
       Alertmanager
-    digest: 5a382220965c57d384b9a66cf09606ec76219242d5b48562c270e5c9f9a5e327
+    digest: f76948784dce142ca88002a9a6a3af1d22c5f91079d1330cb14ebb87c2cbea93
+    icon: https://charts.rancher.io/assets/logos/alerting-drivers.svg
```

Comment:
Your changes should be nothing about the prom2teams chart. 
I guess maybe your index.yaml is from a wrong branch?

Reply:
https://github.com/rancher/charts/blob/941bef117c51d9d838678a3f97f9411428134d0e/charts/rancher-alerting-drivers/rancher-alerting-drivers/1.0.200/Chart.yaml#L16 so it looks like the icon is infact in the chart, so the index.yaml must've had old data, so when I re-did it, it corrected

Reply:
This may be related to comments below, something odd going on with index generation

Reply:
Confirmed that in dev-v2.5 the icon is not on 1.0.100 or 1.0.200. It only got added to 1.0.200 here but thats likely because thats where package currently is.